paket.dependencies

```diff
@@ -3,10 +3,10 @@ framework: auto-detect
 source http://nuget.org/api/v2
 
 nuget FSharp.Core >= 4.3.4 lowest_matching:true
-nuget Expecto
-nuget Expecto.FsCheck
-nuget FsCheck 
-nuget Expecto.VisualStudio.TestAdapter version_in_path: true
+nuget Expecto 8.12
```

Comment:
Versions added to stay on the versions used in the previous version - I didn't want to change the versions.